hooks/boot/05-environment.rb

```diff
@@ -1,5 +1,5 @@
 %w[http_proxy https_proxy ssl_cert_file HTTP_PROXY HTTPS_PROXY SSL_CERT_FILE].each do |variable|
-  if ::ENV.delete(variable)
+  if ENV.delete(variable)
```

Comment:
RuboCop is wrong here, and it needs to be `::ENV` so it's not interpreted as `Kafo::ENV`. See https://github.com/theforeman/kafo/blob/a1b1b12d47f2e0dd26629452809c915e40a861b4/lib/kafo/kafo_configure.rb#L3-L8

Reply:
@ekohl Thanks! I've fixed it now and it is ready for the review.